commcare_connect/users/helpers.py

```diff
@@ -37,6 +39,7 @@ def create_hq_user(user, domain, api_key):
     )
     if hq_request.status_code == 201:
         return True
+    logging.info(f"Create User Response: {hq_request.text}")
```

Comment:
Maybe this should be `logging.error` - alternatively what about raising an exception and capturing it in Sentry?

Reply:
I added httpx and then raised an exception on raise for status. Since we are only interested any errors which are not 2xx response codes. https://github.com/dimagi/commcare-connect/pull/196/commits/28e2b999ebd8abb6924121fd675cb427928b50d9